tkltest/generate/unit/generate.py

```diff
@@ -180,7 +180,14 @@ def generate_ctd_amplified_tests(config, output_dir):
 
     if os.path.exists(test_directory):
         shutil.rmtree(test_directory)
-    shutil.move(tmp_test_directory, test_directory)
+    # todo - resolve at core, and remove this if (leave just the else)
+    if os.path.isdir(os.path.join(tmp_test_directory, 'monolithic')):
+        shutil.move(os.path.join(tmp_test_directory, 'monolithic'), test_directory)
+        for file in os.listdir(tmp_test_directory):
+            shutil.move(os.path.join(tmp_test_directory, file), test_directory)
+        shutil.rmtree(tmp_test_directory)
+    else:
```

Comment:
When will this else branch be satisfied? seems like never. If that is indeed the case, please remove the if-else